.travis.yml

```diff
@@ -22,6 +27,13 @@ install:
 
 before_script:
   - psql -c 'create database mydb;' -U postgres
+  # Add authentication to postgresql
+  - psql -U postgres -c "ALTER USER postgres WITH PASSWORD '${DB_PASSWORD}';"
+  - sudo rm /etc/postgresql/$PGVERSION/main/pg_hba.conf
```

Comment:
Because by default the config is `trust` anything coming from localhost. If we do that, then any password would be accepted, and if any password is accepted, we cannot test the behaviour of the rds broker when the password is wrong. 

The real alternative is not to test the wrong password in the unit tests at all.

Reply:
Does this mean that we could still have an empty password?